src/client/components/HealthCareApp.jsx

```diff
@@ -144,32 +154,68 @@ class HealthCareApp extends React.Component {
           afterText="»"/>
     );
 
-    const submitButton = (
-      <ProgressButton
-          onButtonClick={this.handleSubmit}
-          buttonText="Submit Application"
-          buttonClass="usa-button-primary"/>
-    );
+    if (submissionStatus === false) {
+      submitButton = (
+        <ProgressButton
+            onButtonClick={this.handleSubmit}
+            buttonText="Submit Application"
+            buttonClass="usa-button-primary"/>
+      );
+    } else if (submissionStatus === 'submitPending') {
+      submitButton = (
+        <ProgressButton
+            onButtonClick={this.handleSubmit}
+            buttonText="Sending..."
+            buttonClass="usa-button-disabled"/>
+      );
+    } else if (submissionStatus === 'applicationSubmitted') {
+      submitButton = (
+        <ProgressButton
+            onButtonClick={this.handleSubmit}
+            buttonText="Submitted"
+            buttonClass="hca-button-green"
+            beforeText="&#10003;"/>
+      );
+    } else {
+      submitButton = (
+        <ProgressButton
+            onButtonClick={this.handleSubmit}
+            buttonText="Send Failed"
+            buttonClass="usa-button-secondary"
+            beforeText="x"/>
+      );
+    }
 
-    if (path === '/review-and-submit') {
+    if (this.props.location.pathname === '/review-and-submit') {
       buttons = (
         <div className="row progress-buttons">
           <div className="small-6 medium-5 columns">
             {backButton}
           </div>
-          <div className="small-6 medium-5 end columns">
+          <div className="small-6 medium-5 columns">
             {submitButton}
           </div>
+          <div className="small-1 medium-1 end columns">
+            <div className={this.state ? 'spinner' : 'hidden'}>&nbsp;</div>
+          </div>
         </div>
       );
-    } else if (path === '/introduction') {
+    } else if (this.props.location.pathname === '/introduction') {
       buttons = (
         <div className="row progress-buttons">
           <div className="small-6 medium-5 columns">
             {nextButton}
           </div>
         </div>
       );
+    } else if (this.props.location.pathname === '/submit-message') {
+      buttons = (
+        <div className="row progress-buttons">
+          <div className="small-6 medium-5 columns">
+            <button className="usa-button-primary">Back to Main Page</button>
```

Comment:
No this was something that I was hoping @EmilyTav and @staceylanger would comment on in our walk-through. I wasn't sure what we wanted the user to do once they have completed and where it should take them. What does "Main Page" mean? Is that `vets.gov` or `vets.gov/health-care/`? Would love thoughts.